pkg/operator/configobserver/node/observe_latency_profile.go

```diff
@@ -56,17 +55,32 @@ func (l *latencyProfileObserver) observeLatencyProfile(
 	existingConfig map[string]interface{},
 ) (ret map[string]interface{}, errs []error) {
 
-	// traverse list of latencyConfigs to get all the config paths
-	configPaths := make([][]string, len(l.latencyConfigs))
-	for i, latencyConfig := range l.latencyConfigs {
-		configPaths[i] = latencyConfig.ConfigPath
-	}
-
 	defer func() {
 		// Prune the observed config so that it only contains fields specific to this observer
-		ret = configobserver.Pruned(ret, configPaths...)
+		ret = configobserver.Pruned(ret, l.usedConfigPaths...)
 	}()
 
+	// at day-0 before the config observers begin writing to observedConfig,
+	// value of existingConfig is empty
+	// run the suppressor(s) only if it's not day-0; because any latency profile can be transitioned at day-0
+	if !reflect.DeepEqual(existingConfig, map[string]interface{}{}) {
```

Comment:
As per issues described in [this comment](https://github.com/openshift/library-go/pull/1372/#issuecomment-1145732452):
- if `existingConfig` is an empty map, we can assume day-0 and not fire the suppressor(s) at all [1]:
    - ensures: no rejection at day-0, when user sets `workerLatencyProfile: Low..` in their bootstrap config-node manifest
    - ensures: `config-0 config map not found` error to not result in `ConfigObservationDegraded` 

[1] https://github.com/openshift/library-go/blob/master/pkg/operator/configobserver/config_observer_controller.go#L130-L133 - it is empty when operator just starts, since `observedConfig` in the operator resource is blank at that time

Reply:
Yes, updated the suppressor(s) to handle revision-0 differently.
Also, https://github.com/openshift/library-go/pull/1372#discussion_r893700908.

Reply:
Follow-up: https://github.com/openshift/library-go/pull/1372/#discussion_r891626900, https://github.com/openshift/library-go/pull/1372/#discussion_r897117146.